test/storage.test.js

```diff
@@ -36,8 +37,10 @@ describe('storage.js', () => {
         afterEach( () => {
             nock.cleanAll();
             mockFs.restore();
+            mockRequire.stopAll();
             delete process.env.WORKER_TEST_MODE;
             delete process.env.ASSET_COMPUTE_DISABLE_RETRIES;
+            process.env.__OW_NAMESPACE = "namespace";
```

Comment:
you should remove this variable after each test:
```
delete process.env.__OW_NAMESPACE;
```
and re-set it before each test

Reply:
Removed it completely not required, this was required when I ran the actual test to azure to generate preSignedUrl for data-uri, which gave the end to end aspect